File: qc-domain/src/main/java/com/eager/questioncloud/domain/post/PostCommentJpaRepository.java
package com.eager.questioncloud.domain.post;

import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;

public interface PostCommentJpaRepository extends JpaRepository<PostCommentEntity, Long> {
    Optional<PostCommentEntity> findByIdAndWriterId(Long id, Long userId);
}

File: qc-community/src/main/kotlin/com/eager/questioncloud/post/infrastructure/repository/PostCommentJpaRepository.kt
package com.eager.questioncloud.post.infrastructure.repository

import com.eager.questioncloud.post.infrastructure.entity.PostCommentEntity
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface PostCommentJpaRepository : JpaRepository<PostCommentEntity, Long> {
    fun findByIdAndWriterId(id: Long, userId: Long): Optional<PostCommentEntity>
}
